src/NServiceBus.RabbitMQ/Configuration/RabbitMQTransportSettingsExtensions.cs

```diff
@@ -128,5 +128,13 @@ public static TransportExtensions<RabbitMQTransport> SetClientCertificates(this
             transportExtensions.GetSettings().Set(SettingsKeys.ClientCertificates, clientCertificates);
             return transportExtensions;
         }
+
+        /// <summary>
+        /// Gets the <see cref="BasicDeliverEventArgs"/> for the current message being processed.
+        /// </summary>
+        public static BasicDeliverEventArgs GetBasicDeliverEventArgs(this IMessageHandlerContext context)
```

Comment:
i figured it want worth leaking a new type for one method when they will never see that class anyway